packages/graphql/src/graphql/directives/type-dependant-directives/authorization.ts

```diff
@@ -191,6 +211,43 @@ function createJWTPayloadWhere(JWTPayloadDefinition?: ObjectTypeDefinitionNode):
     return inputTC.getType();
 }
 
+function getJwtStandardFields(schema: GraphQLSchema) {
+    const jwtStandardType = new GraphQLObjectType({
+        name: "JWTStandard",
+        fields: {
```

Comment:
This doesn't seem to be all of the standard fields listed in https://github.com/neo4j/graphql/blob/dev/docs/rfcs/rfc-036-authorization.md#jwt-payload - is there a reason for this?

Reply:
no reason, wrong reference of standard fields. They should all be there now.